recipes/xqilla/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+required_conan_version = ">=1.40.0"
+
+
+class ConanXqilla(ConanFile):
+    name = "xqilla"
+    
+    description = (
+        "XQilla is an XQuery and XPath 2 library and command line utility written in C++, implemented on top of the Xerces-C∞ library"
+    )
+    topics = ("conan", "xqilla", "XML", "XQuery")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://xqilla.sourceforge.net/HomePage"
+    license = "Apache-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    exports_sources = "patches/**"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("xerces-c/3.2.3")
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("xqilla is just supported for Linux")
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True,
+                  destination=self._source_subfolder)
+
+    def _configure_autotools(self):
+        if self._autotools:
+            return self._autotools
+        self._autotools = AutoToolsBuildEnvironment(self)
+        conf_args = [
+            "--with-xerces={}".format(tools.unix_path(self.deps_cpp_info["xerces-c"].rootpath)),
+            "CXXFLAGS=-std=c++11",
```

Comment:
This will override `CXXFLAGS` from a recipe.
Better is to do:
```python
if not self.settings.compiler.cppstd:
    self._autotools.cppstd_flag = "-std=c++11"
```

Reply:
Updated.